src/containers/Configure/Configure.js

```diff
@@ -15,124 +15,127 @@ const Configure = ({userQuote, allUserQuotes}) => {
   const [error, updateError] = useState('')
   const [formCompleted, updateFormCompleted] = useState(false)
 
-useEffect(() =>{
-  // setIsLoading(true)
-  fetch('https://developer.nrel.gov/api/pvwatts/v6.json?api_key=By8qOhq8GrFH18lkeImHNhinPb7jIbCbibKlQNsS&address=4521A S Crystal Way Aurora CO 80015&system_capacity=4&azimuth=180&tilt=20&array_type=0&module_type=0&losses=14')
-  .then(response => {
-    if (!response.ok){
-      throw new Error('Failed to fetch')
+  useEffect(() => {
+    // setIsLoading(true)
+    fetch('https://developer.nrel.gov/api/pvwatts/v6.json?api_key=By8qOhq8GrFH18lkeImHNhinPb7jIbCbibKlQNsS&address=4521A S Crystal Way Aurora CO 80015&system_capacity=4&azimuth=180&tilt=20&array_type=0&module_type=0&losses=14')
```

Comment:
This will be updated in a later PR to point to the BE server with environmental variables